Source/Core/DolphinWX/CMakeLists.txt

```diff
@@ -212,6 +212,10 @@ if(wxWidgets_FOUND)
 		install(DIRECTORY ${BUNDLE_PATH} DESTINATION /Applications
 			USE_SOURCE_PERMISSIONS
 			)
+	elseif(WIN32)
+		set_target_properties(${DOLPHIN_EXE} PROPERTIES
+			WIN32_EXECUTABLE ON
```

Comment:
I mean that the `WIN32_EXECUTABLE` has no effect when CMake isn't already targeting WIN32